pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-advanced/aa-bootstrap/index.md

```diff
@@ -23,7 +23,7 @@ Before you begin, you must:
 1.  Create a bootstrap cluster:
 
     ```bash
-    dkp create bootstrap --kubeconfig $HOME/.kube/config
+    dkp create bootstrap --with-aws-bootstrap-credentials=true --kubeconfig $HOME/.kube/config
```

Comment:
Not sure If I understood this the wrong way, but the command mentions AWS (to use AWS credentials), but in this file we are talking about AKS infrastructure? Should that not be AKS credentials?

Reply:
Good catch, I meant to add this to EKS 🙇